### PR TITLE
Card Client Docs

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - Add Docs for iOS Card Client Integration

 ### Checklist

 ~- [ ] Added a changelog entry~

### Authors

- @minhthenguyen
- @sshropshire

### CODE REVIEW

**Comment by sshropshire** [comment]
See [iOS-SDK #29](https://github.com/paypal/iOS-SDK/pull/29)

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The In-App payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
+
+You need to have a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK will allow you to approve the order on the client side of your app by using the buyer's card information.
```
Comment (sarahkoop) [inline docs/index.md:9]:
```suggestion
You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). The Payments SDK allows for client-side approval of an order with the user's card information.
```
Just some potential wording suggestions

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+# Pay with Card
+
+Accept card payments in your app using the PayPal In-App Payments SDK.
+
+## How it works
+
+The In-App payments SDK allows for merchant apps to integrate easily with PayPal payment services on both the client and server.
```
Comment (sarahkoop) [inline docs/index.md:7]: We aren't super consistent with the naming of the SDK - is In-App Payments SDK the name or just Payments SDK? Either way we should make the capitalization and naming the same across this doc.

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+### Steps
+
+#### 1. Create an order
+Call `v2/checkout/orders` to create an order an obtain an order ID
+
+```
+curl --location --request POST 'https://api.sandbox.paypal.com/v2/checkout/orders/' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer <access_token>' \
```
Comment (sarahkoop) [inline docs/index.md:46]: Should we include docs here for how to obtain the access token or will those live somewhere else?

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+}'
+```
+
+#### 2. Approve the order using Payments SDK:
+
+##### iOS
+```swift
+import PaymentsSDK
+
+// Create an config object using your clientID obtained from PayPal application portal
```
Comment (sarahkoop) [inline docs/index.md:66]:
```suggestion
// Create a config object using your clientID obtained from PayPal application portal
```

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+let config = CoreConfig(clientID: "<CLIENT_ID>", evironment: .sandbox)
+
+// Create a CardClient object to process a card payment
```
Comment (sarahkoop) [inline docs/index.md:69]:
```suggestion
// Create a CardClient to approve an order with a card payment method
```

File: docs/index.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+let cardClient = CardClient(config: config)
+
+// Create a card object from buyer's inputs
```
Comment (sarahkoop) [inline docs/index.md:72]: Nitpick but I prefer "user" over "buyer" throughout the docs - does anyone else have on opinion on this?

File: docs/index.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    expirationYear: "25",
+    securityCode: "123"
+)
+
+// Create a CardRequest and approve the order using buyer's card when buyer submit their card info.
+let cardRequest = CardRequest(orderID: "<ORDER_ID>", card: card)
+cardClient.approveOrder(request: cardRequest) { result in
+    switch result {
+        case .success(let result):
+            // Order is successfully approved with card and ready to be capture/authorize.
```
Comment (sarahkoop) [inline docs/index.md:85]:
```suggestion
            // Order was successfully approved with the card and is ready to be captured/authorized.
```

File: docs/index.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+        case .failure(let error):
+            // Encountered error when approving order.
+    }
+}
+```
+
+##### Android
```
Comment (sarahkoop) [inline docs/index.md:92]: Are we going to have this same document in the both the Android and iOS repos? We could remove this section if the Android docs will just live in that repo, or could keep it here to ensure consistency.

**Comment by sarahkoop** [comment]
Sorry ignore all these comments, I moved them to the correct PR in the iOS repo 🤦

**Pull request closed without merging**